pandas/tests/io/formats/test_to_html.py

```diff
@@ -1844,6 +1844,64 @@ def test_to_html_no_index_max_rows(self):
         </table>""")
         assert result == expected
 
+    def test_to_html_pivot_table_max_cols(self):
+        # GH https://github.com/pandas-dev/pandas/issues/6131
+        df = DataFrame([
+            {'a': 'aa', 'b': 'ba', 'c': 'ca', 'd': 1},
+            {'a': 'ab', 'b': 'bb', 'c': 'cc', 'd': 2},
+            {'a': 'ac', 'b': 'bc', 'c': 'cc', 'd': 3}
+        ])
+        result = df.pivot_table(
```

Comment:
Rather than creating `df` and then pivoting can you simplify and just create the desired frame (i.e. the result of the pivot)?